src/cpp/session/modules/SessionDataViewer.R

```diff
@@ -249,13 +249,14 @@
     {
       # create a temporary frame to hold the value; this is necessary because
       # "x" is a function argument and therefore a promise whose value won't
-      # be bound via substitute() below
-      coerced <- x
+      # be bound via substitute() below. we use a random-looking name so we 
+      # can spot it later when relabeling columns.
+      ioHe4iej <- x
```

Comment:
As extra safety, we could make this a non-syntactic name e.g. 

    __RSTUDIO_VIEWER_COLUMN__

Reply:
👍